src/github.com/getlantern/balancer/balancer_test.go

```diff
@@ -227,11 +230,9 @@ func TestAll(t *testing.T) {
 	}
 
 	// Test failure
-	b = New(dialer4)
+	b = New(dialer4, dialer4)
```

Comment:
I don't know exactly the original purpose of this test, but it failed on my Mac even on branch valencia. I revised it to show that balancer can retry dialling on all qualified dialers. It would be good to point out my misunderstanding on it.